configs/default/VGRC-VGOODRCF4.config

```diff
@@ -57,70 +57,82 @@ resource USB_DETECT 1 C04
 # timer
 timer C07 AF3
 # pin C07: TIM8 CH2 (AF3)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
 timer A09 AF1
 # pin A09: TIM1 CH2 (AF1)
-timer B00 AF2
-# pin B00: TIM3 CH3 (AF2)
-timer B01 AF2
-# pin B01: TIM3 CH4 (AF2)
+timer A10 AF1
+# pin A10: TIM1 CH3 (AF1)
 timer C08 AF3
 # pin C08: TIM8 CH3 (AF3)
 timer C09 AF3
 # pin C09: TIM8 CH4 (AF3)
-timer A08 AF1
-# pin A08: TIM1 CH1 (AF1)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
 
 # dma
-dma ADC 2 1
-# ADC 2: DMA2 Stream 3 Channel 1
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
 dma pin C07 0
 # pin C07: DMA2 Stream 2 Channel 0
-dma pin A09 0
-# pin A09: DMA2 Stream 6 Channel 0
+dma pin A08 1
+# pin A08: DMA2 Stream 1 Channel 6
+dma pin A09 1
+# pin A09: DMA2 Stream 2 Channel 6
+dma pin A10 1
+# pin A10: DMA2 Stream 6 Channel 6
+dma pin C08 1
+# pin C08: DMA2 Stream 4 Channel 7
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
 dma pin B00 0
 # pin B00: DMA1 Stream 7 Channel 5
 dma pin B01 0
 # pin B01: DMA1 Stream 2 Channel 5
-dma pin C08 0
-# pin C08: DMA2 Stream 2 Channel 0
-dma pin C09 0
-# pin C09: DMA2 Stream 7 Channel 7
-dma pin A08 1
-# pin A08: DMA2 Stream 1 Channel 6
 
 # feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature LED_STRIP
 feature OSD
 
+# serial
+serial 5 64 115200 57600 0 115200
+
+# led
+led 0 0,15::ATI:0
+led 1 1,15::ATI:0
+led 2 2,15::ATI:0
+led 3 3,15::ATI:0
+led 4 4,15::ATI:0
+led 5 5,15::ATI:0
+led 6 6,15::ATI:0
+
+# aux
+aux 0 0 0 1700 2100 0 0
+
 # master
-set gyro_to_use = FIRST
-set align_mag = DEFAULT
 set mag_bustype = I2C
 set mag_i2c_device = 1
-set mag_i2c_address = 0
-set mag_spi_device = 0
 set baro_bustype = I2C
-set baro_spi_device = 0
 set baro_i2c_device = 2
-set baro_i2c_address = 0
-set adc_device = 2
+set serialrx_provider = SBUS
 set blackbox_device = SPIFLASH
-set dshot_burst = OFF
+set motor_pwm_protocol = DSHOT600
```

Comment:
If this setting is used as a default, and if your board is used in combination with older ESCs that do not support Dshot, this will result in the motors spinning up as soon as a battery is connected, potentially resulting in damage or injury.
Do you assume all responsibility for personal injury and damage resulting from using this default setting?

Reply:
Thanks for your kind remind for the safety.
We sell this controller with our latest BLHeli_32 ESC and our ESC default also with Dshot600, at the same time, we will write and enhance this safety instruction in our FC manual. So we would to keep this default value as this, this is also required by our customer. Thanks for it.